cime_config/config_component_cesm.xml

```diff
@@ -241,6 +241,10 @@
       <!-- High res BLOM -->
       <!-- =================================================== -->
       <value compset="_BLOM" grid="oi%tnx0.25v">48</value>
+      <!-- =================================================== -->
+      <!-- High res WW3DEV -->
+      <!-- =================================================== -->
+      <value compset="_WW3DEV" grid="w%w025">72</value>
```

Comment:
This can be used for testing but is incompatible with many coupled compsets (e.g., 1/2 degree needs 96, higher-top models often need shorter intervals). I think this section should (eventually) be expanded with more entries for different atmosphere (and ocean) configurations and resolutions. Perhaps a note about these plans would help.

Reply:
Just added this comment. Thanks!